internal/cmd/skupper/connector/nonkube/connector_delete_test.go

```diff
@@ -147,17 +145,40 @@ func TestCmdConnectorDelete_Run(t *testing.T) {
 
 		t.Run(test.name, func(t *testing.T) {
 
+			createConnectorResource(path, t)
 			cmd.connectorName = test.deleteName
 			cmd.namespace = test.namespace
 			cmd.connectorHandler = fs.NewConnectorHandler(cmd.namespace)
 			cmd.InputToOptions()
 
 			err := cmd.Run()
 			if err != nil {
-				assert.Check(t, test.errorMessage == err.Error())
+				assert.Check(t, strings.HasSuffix(err.Error(), test.errorMessage), err.Error())
 			} else {
 				assert.Check(t, err == nil)
 			}
 		})
 	}
 }
+
+func createConnectorResource(path string, t *testing.T) {
+	connectorResource := v2alpha1.Connector{
+		TypeMeta: metav1.TypeMeta{
+			APIVersion: "skupper.io/v2alpha1",
+			Kind:       "Connector",
+		},
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      "my-connector",
+			Namespace: "test",
+		},
+	}
+
+	connectorHandler := fs.NewConnectorHandler("test")
+
+	defer connectorHandler.Delete("my-connector")
```

Comment:
I would suggest to assert the returned value for the `Delete` method.

Reply:
done, removed deferred delete function given that it is using a temp dir.